internal/controller/apiextensions/composite/composition_pt.go

```diff
@@ -45,7 +45,7 @@ import (
 const (
 	errGetComposed   = "cannot get composed resource"
 	errGCComposed    = "cannot garbage collect composed resource"
-	errApplyComposed = "cannot apply composed resource"
+	errApplyComposed = "cannot apply composed resource %q"
```

Comment:
Can you rename this and move it down to the block below with all the other `errFmt` strings, please.

 
```suggestion
	errFmtApplyComposed = "cannot apply composed resource %q"
```